docs/sdks/client-sdks/android.md

```diff
@@ -13,7 +13,7 @@ Eppo's open source Android SDK can be used for both feature flagging and experim
 You can install the SDK using Gradle by adding to your `build.gradle` file:
 
 ```groovy
-implementation 'cloud.eppo:android-sdk:3.2.0'
+implementation 'cloud.eppo:android-sdk:3.+'
```

Comment:
FYI it's not recommended to use `+` syntax on Gradle as it leads to non-deterministic behavior with unexpected changing library versions. I know this is a different convention than what folks usually do for NPM/JS ecossystem, for example, but on JVM world people are a lot more conscious of this stuff. Just wanted to call that out

Reply:
Just a data point:
<img width="1059" alt="image" src="https://github.com/user-attachments/assets/5310c491-31b8-46c1-b128-bf0a2dd743e6">
https://developer.vonage.com/en/blog/the-right-way-of-using-gradle-dynamic-dependencies

Reply:
Good callout - I didn't know. Reverting to point explicitly to the latest version.

Reply:
thank you 🙏🏽